src/mesh/tools.rs

```diff
@@ -625,54 +622,45 @@ mod tests {
         Mesh::from_triangle_faces_with_vertices_and_normals(faces, vertex_positions, vertex_normals)
     }
 
-    pub fn welded_box_smooth_mesh(position: [f32; 3], scale: f32) -> Mesh {
-        let vertex_positions = vec![
-            // back
-            v(-1.0, 1.0, -1.0, position, scale),
-            v(-1.0, 1.0, 1.0, position, scale),
-            v(1.0, 1.0, 1.0, position, scale),
-            v(1.0, 1.0, -1.0, position, scale),
-            // front
-            v(-1.0, -1.0, -1.0, position, scale),
-            v(1.0, -1.0, -1.0, position, scale),
-            v(1.0, -1.0, 1.0, position, scale),
-            v(-1.0, -1.0, 1.0, position, scale),
+    pub fn welded_box_smooth_mesh() -> Mesh {
+        let vertices = vec![
+            Point3::new(1.0, 1.0, 1.0),
+            Point3::new(-1.0, -1.0, 1.0),
+            Point3::new(-1.0, 1.0, 1.0),
+            Point3::new(1.0, 1.0, -1.0),
+            Point3::new(-1.0, 1.0, -1.0),
+            Point3::new(-1.0, -1.0, -1.0),
+            Point3::new(1.0, -1.0, 1.0),
+            Point3::new(1.0, -1.0, -1.0),
         ];
 
-        let vertex_normals = vec![
-            n(-1.0 / 3.0, 1.0 / 3.0, -1.0 / 3.0),
-            n(-1.0 / 3.0, 1.0 / 3.0, 1.0 / 3.0),
-            n(1.0 / 3.0, 1.0 / 3.0, 1.0 / 3.0),
-            n(1.0 / 3.0, 1.0 / 3.0, -1.0 / 3.0),
-            // front
-            n(-1.0 / 3.0, -1.0 / 3.0, -1.0 / 3.0),
-            n(1.0 / 3.0, -1.0 / 3.0, -1.0 / 3.0),
-            n(1.0 / 3.0, -1.0 / 3.0, 1.0 / 3.0),
-            n(-1.0 / 3.0, -1.0 / 3.0, 1.0 / 3.0),
+        let normals = vec![
+            n(0.25, 0.5, 0.25),
```

Comment:
Since you are already doing cleanup, could you please remove the `n` function in this file? I think we can just use `Vector3::new`

Reply:
ok